task-2/src/components/MoviesList/index.js

```diff
@@ -1,32 +1,52 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
 
 import PropTypes from 'prop-types';
-import { useDispatch } from 'react-redux';
 
-import { movieData } from '../../core/data';
-import { switchBanner } from '../../store/actionCreators/actionCreators';
+import axios from 'axios';
+
+import { switchBanner, allLoadedMovies } from '../../store/actionCreators';
 
 import { Movie } from '../Movie';
 
 import './index.css';
 
+const movieApi = 'http://localhost:4000/movies?limit=10';
```

Comment:
Consider creating a separate folder "api" at "src/core" level where you'd keep such constants along with anything related to making API calls.

Reply:
Fixed

Reply:
fixed.

Reply:
Left a comment below.